history-of-the-quakers-v4/original/11-book-7-chapter-5.adoc

```diff
@@ -411,6 +428,9 @@ for Easter-offerings and small tithes, at the suit of John Wright vicar of Stepn
 The rates set upon Easter-offerings, and tithes claimed by the said John Wright,
 in his bill of complaint against them preferred in the Exchequer, were as followeth,
 
+[.small-break]
```

Comment:
This is another chart...I did this before talking to you about it. I thought it looked good in the PDF but I'm sure we can use the chart when that's ready.

Reply:
ok, thanks! i'm happy leaving it for now.